app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/dashboard.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
lets not disable so important lints globally in the file, please. Or we need to do it because of `.tsx` and react? If the latter, is it possible to configure the linter to have these rules disabled for all `tsx` files? EDIT: this is probably not a good idea as I see other `tsx` files that do not disable these lints.

Reply:
These were necessary because a hastily-written version of the dashboard was added at first. These lints where disabled temporarily for development.